src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -106,9 +108,9 @@
   @SerializedName("last_payment_error")
   StripeError lastPaymentError;
 
-  /** The [Level III data](/docs/level3) associated with this payment. */
+  /** The [Level III data](https://stripe.com/docs/level3) associated with this payment. */
   @SerializedName("level3")
-  Level3 level3;
+  Charge.Level3 level3;
```

Comment:
I still believe we're making a mistake adding this in stripe-java. We should not support this until it's stable. Payflows used this like Charge but I believe they will change this. Not a _huge deal_ but flagging again anyway